cachi2/core/utils.py

```diff
@@ -1,20 +1,25 @@
+import errno
 import json
 import logging
 import os
 import re
 import shutil
 import subprocess  # nosec
+import sys
+from functools import cache
 from pathlib import Path
 from typing import Callable, Iterator, Optional, Sequence
 
-import reflink  # type: ignore
-
 from cachi2.core.config import get_config
 from cachi2.core.errors import Cachi2Error
 
 log = logging.getLogger(__name__)
 
 
+class _FastCopyFailedFallback(Exception):
+    """Fast in-kernel copying failed fallback to regular copy."""
```

Comment:
It could be just me, but the sentence reads a bit hard. If you will be making another revision I'd suggest either a semicolon after 'failed' ("copying failed: fallback to regular copy") or maybe just "Signals fall back from fast-in kernel copying to regular copy"?